.github/workflows/build-test-push.yaml

```diff
@@ -63,12 +61,15 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         run: |
           echo IMAGE_TAG="$GITHUB_REF_NAME" >> "$GITHUB_ENV"
+      - name: Tag image
+        if: env.IMAGE_TAG != ''
+        run: |
+          docker tag ${{ env.SOURCE_IMAGE_NAME }} ${{ env.GAR_IMAGE_PATH }}:${{ IMAGE_TAG }}
       - name: Push Docker image to GAR
         if: env.IMAGE_TAG != ''
         uses: mozilla-it/deploy-actions/docker-push@v4.0.0
         with:
-          local_image: ${{ env.IMAGE_NAME }}
-          image_repo_path: ${{ env.IMAGE_REPO_PATH }}
-          image_tag: ${{ env.IMAGE_TAG }}
-          workload_identity_pool_project_number: ${{ vars.GCPV2_WORKLOAD_IDENTITY_POOL_PROJECT_NUMBER }}
+          gar_location: us
```

Comment:
You should be able to safely remove this input since `us` is the default

Reply:
The docs say this is required and it was in the example, so that's why I added it.

I'll remove it. Thanks!